### ISSUE TITLE
Odd behavior with New Inherited Scene

### ISSUE THREAD

**Issue by GinghamJones** [godotengine/godot#72103]
### Godot version

4.0 Beta 15

### System information

Windows 11

### Issue description

When creating a new inherited scene, it's like Godot is only opening the parent scene in another tab. Sometimes Godot crashes after saving and switching to the parent scene. Both scenes are then corrupted on exited and reopening the project. 

### Steps to reproduce

1: Create a new project
2: Create scene with root Node3D and save
3: Make new inherited scene from Node3D scene, rename it, then save
4: Close Godot and reopen the project

### Minimal reproduction project

N/A

**Comment by akien-mga** [comment]
CC @KoBeWi, and @SaracenOne in case it's related to recent reload changes.

**Comment by nezvers** [comment]
I just encountered the same problem.

1. Create and save a scene.
2. Scene/New Inherited Scene from the base scene
 In the scene tab, both scenes show the same path.
3. Saving inherited scene doesn't prompt where to save.
4. Opening the only scene shows `Error while loading file 'filename.tscn'.`

Also:
1. After creating a base scene, save and close the tab.
2. Right-click in FileSystem/New Inherited Scene opens without any prompt.
3. Trying to open the base scene regularly in parallel does nothing.

**Comment by eh-jogos** [comment]
Just ran into this bug right now as well. From what I can gather from git what is happening is that the inherited scene is created with the same path as the original scene, so saving will overwrite the original scene and corrupt it, because it will be trying to inherit itself. 

So it's like @nezvers says, you have to create a new inherited scene and Save Scene As so that you don't corrupt the original, but this is really odd.

**Comment by akien-mga** [comment]
Might be a duplicate of #72003, in which case it should be fixed in the `master` branch.

**Comment by SaracenOne** [comment]
Yeah, try this again on master. I just tested it now, and I can't reproduce the same problem. I think this was likely the original issue I patched, but let me know if the issue still persists on the latest master.

**Comment by KoBeWi** [comment]
No answer, so assuming it's fixed, based on the above comment.
